open-metadata-implementation/common-services/generic-handlers/src/main/java/org/odpi/openmetadata/commonservices/generichandlers/SchemaAttributeHandler.java

```diff
@@ -717,4 +720,37 @@ public SCHEMA_ATTRIBUTE getSchemaAttribute(String userId,
 
         return this.getSchemaAttributesFromEntities(userId, schemaAttributeEntities, methodName);
     }
+
+    /**
+     * Update a schema attribute
+     * @param userId                      calling user
+     * @param externalSourceGUID          unique identifier of software server capability representing the caller
+     * @param externalSourceName          unique name of software server capability representing the caller
+     * @param schemaAttributeGUID         unique identifier of schema attribute
+     * @param instanceProperties          the schema attribute's properties
+     * @throws InvalidParameterException  one of the parameters is invalid
+     * @throws UserNotAuthorizedException the user is not authorized to issue this request
+     * @throws PropertyServerException    there is a problem reported in the open metadata server(s)
+     */
+    public void updateSchemaAttribute(String userId,
+                                      String externalSourceGUID,
+                                      String externalSourceName,
+                                      String schemaAttributeGUID,
+                                      InstanceProperties instanceProperties) throws InvalidParameterException,
+                                                                                    PropertyServerException,
+                                                                                    UserNotAuthorizedException {
```

Comment:
Code layout (method parameters and first brace)

Reply:
I missed that one :) now it should be fixed